README.md

```diff
@@ -1,26 +1,26 @@
-# Dependencies:
+# London Hackspace ACNode Server
 
-Assuming Debian jessie:
+## Development Environment
 
-```
-apt-get install python-django python-django-auth-ldap python-sqlite python-psycopg2 python-netaddr python-pip
-```
+Copy `acserver/ldap_local_settings_example.py` to `local_settings.py` remove the DB configuration and run `docker-compose up` for a functional development environment
 
-Also and/or otherwise try running `make`, the included Makefile should build a virtualenv with the right bits in it for you. You can activate the virtualenv with `. ./venv/bin/activate`
-
-## For LDAP syncing:
+To create an initial user for testing:
 
 ```
-pip install --verbose --no-deps django-ldap-sync
+docker-compose exec server python manage.py createsuperuser
 ```
 
-To setup the db run:
+## Legacy
```

Comment:
I really don't like calling things legacy when they're just an alternative method the author didn't like :-P

I think in many cases, just using virtualenv will be enough - I don't really see what Docker adds to this. Nothing wrong with it being an option, but I don't think I'd call it "legacy"